src/rules/utils.ts

```diff
@@ -713,96 +725,193 @@ export const isTestCaseCall = (
 
   if (
     callee.type === AST_NODE_TYPES.MemberExpression &&
-    isTestCaseProperty(callee.property)
+    isSupportedAccessor(callee.property) &&
+    properties.includes(getAccessorValue(callee.property))
   ) {
     // if we're an `each()`, ensure we're the outer CallExpression (i.e `.each()()`)
     if (
       getAccessorValue(callee.property) === 'each' &&
       node.callee.type !== AST_NODE_TYPES.TaggedTemplateExpression &&
       node.callee.type !== AST_NODE_TYPES.CallExpression
     ) {
-      return false;
+      return null;
     }
 
-    return callee.object.type === AST_NODE_TYPES.MemberExpression
-      ? isTestCaseName(callee.object.object)
-      : isTestCaseName(callee.object);
+    const nod =
+      callee.object.type === AST_NODE_TYPES.MemberExpression
+        ? callee.object.object
+        : callee.object;
+
+    if (isSupportedAccessor(nod)) {
+      return getAccessorValue(nod);
+    }
   }
 
-  return false;
+  return null;
 };
 
-const isDescribeAlias = (node: TSESTree.LeftHandSideExpression) =>
-  node.type === AST_NODE_TYPES.Identifier &&
-  DescribeAlias.hasOwnProperty(node.name);
-
-const isDescribeProperty = (
-  node: TSESTree.Expression | TSESTree.PrivateIdentifier,
-): node is AccessorNode<DescribeProperty> =>
-  isSupportedAccessor(node) &&
-  DescribeProperty.hasOwnProperty(getAccessorValue(node));
-
 /**
  * Checks if the given `node` is a *call* to a `describe` function that would
  * result in a `describe` block being created by `jest`.
  *
  * Note that `.each()` does not count as a call in this context, as it will not
  * result in `jest` creating any `describe` blocks.
- *
- * @param {TSESTree.CallExpression} node
- *
- * @return {node is JestFunctionCallExpression<TestCaseName>}
  */
 export const isDescribeCall = (
   node: TSESTree.CallExpression,
+  scope: TSESLint.Scope.Scope,
 ): node is JestFunctionCallExpression<DescribeAlias> => {
-  if (isDescribeAlias(node.callee)) {
-    return true;
+  let name = findFirstCallPropertyName(node, Object.keys(DescribeProperty));
+
+  if (!name) {
+    return false;
   }
 
-  const callee =
-    node.callee.type === AST_NODE_TYPES.TaggedTemplateExpression
-      ? node.callee.tag
-      : node.callee.type === AST_NODE_TYPES.CallExpression
-      ? node.callee.callee
-      : node.callee;
+  name = resolveToJestFn(scope, name);
+
+  return !!name && DescribeAlias.hasOwnProperty(name);
+};
+
+interface ImportDetails {
+  source: string;
+  local: string;
+  imported: string;
+}
+
+const describeImportDefAsImport = (
+  def: TSESLint.Scope.Definitions.ImportBindingDefinition,
```

Comment:
```suggestion
  definition: TSESLint.Scope.Definitions.ImportBindingDefinition,
```

Reply:
A little on the fence about this - I went with `def` because that's what the underlying runtime property that's passed in is called